### PR TITLE
Check what will be if I enable `concurrency-mt-unsafe` in `clang-tidy`

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: utils/zookeeper-test/main.cpp [hunk 1]
```diff
@@ -1,364 +0,0 @@
...
-#include <Common/Stopwatch.h>
-
-#include <iostream>
-#include <sstream>
-#include <exception>
-#include <future>
-
-using namespace std;
-
-/// TODO: Remove ME
```
Comment (alexey-milovidov) [inline utils/zookeeper-test/main.cpp:22]: I've implemented this TODO.

File: utils/graphite-rollup/graphite-rollup-bench.cpp [hunk 2]
```diff
@@ -1,130 +0,0 @@
-#include <chrono>
```
Comment (alexey-milovidov) [inline utils/graphite-rollup/graphite-rollup-bench.cpp:1]: Non-automated test, no one use or support it.

File: src/IO/examples/hashing_buffer.h [hunk 3]
```diff
@@ -1,21 +0,0 @@
-#pragma once
```
Comment (alexey-milovidov) [inline src/IO/examples/hashing_buffer.h:1]: Non-automated unit test, most likely unneeded.

File: base/base/errnoToString.cpp [hunk 4]
```diff
@@ -3,10 +3,11 @@
 #include <fmt/format.h>
 
 
-std::string errnoToString(int code, int the_errno)
```
Comment (alexey-milovidov) [inline base/base/errnoToString.cpp:6]: The usage of the "code" argument was strange and misinterpreted by the callers. I've removed it.

**Comment by alexey-milovidov** [comment]
I don't understand the result of Debug tests:

> <Warning> TCPHandler: Client has gone away.

**Review by rschu1ze — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
@kssenii a test for Azure blob storage has failed:
https://s3.amazonaws.com/clickhouse-test-reports/40224/e8a6eb5dd2d16452b51736685cebd888c3287762/integration_tests__asan__[2/3].html

**Pull request merged**